README.md

```diff
@@ -49,20 +49,51 @@ The Upgrader updates projects via one of the following approaches:
 ​
 To enable the Upgrader to execute this step, apply the following configuration:
 ​
+* Enable GitHub source code provider:
```

Comment:
Please mention that the default source code provider is `github` and if it's necessary to switch via supported source code providers you have to export environment variable.

Reply:
Thanks! Changed to `GitHub is default source code provider. If you want manually define it, use the next environment variable:`